playbooks/templates/.env.j2

```diff
@@ -107,64 +133,96 @@ AIRTABLE_FIELD={{ webportal_common_config.airtable_field }}
 # MongoDB
 
 # MongoDB username
-SKYNET_DB_USER={{ mongodb_config.skynet_db_user }}
+#
+# OPTIONAL: manually defined by user, or default will be used
+SKYNET_DB_USER={{ mongodb_config.skynet_db_user | default('admin') }}
 
 # MongoDB password
+#
+# REQUIRED: Ansible automatically generated 
+# TODO: Ansible doesn't not currently automatically generate this but it should 
 SKYNET_DB_PASS={{ mongodb_config.skynet_db_pass }}
 
 # MongoDB address or container name
+#
+# OPTIONAL: manually defined by user, or default will be used
+# TODO: Do these need to be able to be defined by the user?
 SKYNET_DB_HOST=mongo
 
 # MongoDB port
+#
+# OPTIONAL: manually defined by user, or default will be used
+# TODO: Do these need to be able to be defined by the user?
 SKYNET_DB_PORT={{ default_mongo_port }}
 
 # MongoDB replicaset name
+#
+# OPTIONAL: manually defined by user, or default will be used
 SKYNET_DB_REPLICASET={{ webportal_common_config.skynet_db_replicaset | default('skynet') }}
 
 {% if portal_accounts_on %}
 ###############################################################################
 # (optional) Accounts
 
 # Absolute url to the (cluster) portal dashboard.
+#
+# REQUIRED: See top of file 
 SKYNET_DASHBOARD_URL=https://account.{{ cluster_domain }}
 
 # Accounts log level
+#
+# OPTIONAL: manually defined by user, or default will be used
+# TODO: Do these need to be able to be defined by the user?
```

Comment:
All log levels should be configurable by the user. They are very helpful when debugging an issue. If a portal op reaches out to us on Discord and asks us about an issue, one of the first things I would do is ask them for their logs with `level=trace`, so we can see detailed logging.